src/plugin/debugger/dosdebug.c

```diff
@@ -438,32 +438,29 @@ int main (int argc, char **argv)
   struct timeval timeout;
   const char *rp = getenv("XDG_RUNTIME_DIR");
 
+  if (!rp || !rp[0]) {
```

Comment:
Please provide a manual
replacement, like `/var/run`.

Reply:
But it wasn't there before, although the comments suggested it was once.

Reply:
Ok, its just that former exit()
is not seen in the patch (but
is seen in the code under
different condition).